pom.xml

```diff
@@ -1474,5 +1478,54 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>samples</id>
+            <modules>
+                <module>samples</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>rat</id>
+            <modules>
+                <module>integration-tests</module>
+                <module>test-coverage</module>
+                <module>samples</module>
+            </modules>
+            <build>
+                <defaultGoal>apache-rat:check</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.rat</groupId>
+                        <artifactId>apache-rat-plugin</artifactId>
+                        <version>0.12</version>
+                        <!-- only run at the root -->
+                        <inherited>false</inherited>
+                        <executions>
+                            <execution>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>check</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
```

Comment:
Move the rat configuration to pluginManagement, so `mvn apache-rat:check` can be run with or without the `-Prat` profile

Reply:
I add a rat profile because I move some module out of the default build, so the plugin failed to scan them.